src/android/radar.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version='1.0' encoding='utf-8'?>
+<resources>
+</resources>
```

Comment:
can this generated file be added to .gitignore and removed?

Reply:
See my other comment about the strings.xml file